src/main/java/com/google/api/codegen/util/ruby/RubyNameFormatter.java

```diff
@@ -65,7 +65,7 @@ public String qualifiedName(NamePath namePath) {
 
   @Override
   public String packageFilePathPiece(Name name) {
-    return name.toLowerUnderscore();
+    return name.toOriginal().toLowerCase();
```

Comment:
Sorry, probably that's because I'm not familiar with this class, what does the `toOriginal` method do?